.github/workflows/go.yml

```diff
@@ -292,6 +292,59 @@ jobs:
       working-directory: build
       run: ./launcher doctor
 
+  container_exec_testing:
+    name: Container Exec Test
+    runs-on: ubuntu-22.04
+    container: ${{ matrix.container }}
+    strategy:
+      fail-fast: false
+      matrix:
+        container:
+          # See https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources
+          - ubuntu:20.04
+    needs:
+      - version_baseline # version_baseline implies build_linux
+      - build # need the other builds too
+    steps:
+
+    # zstd is needed so we can restore cache -- see https://github.com/actions/cache/issues/1455#issuecomment-2328358604;
+    # ca-certificates and openssl are needed to download osquery.
+    - name: Install dependencies
+      run: |
+        apt-get -y update
+        apt-get -y install zstd ca-certificates openssl
+        update-ca-certificates
+
+    - name: cache restore build output
+      uses: actions/cache/restore@v4
+      with:
+        path: ./build
+        key: ${{ runner.os }}-${{ github.run_id }}
+        enableCrossOsArchive: true
+
+    - name: Launcher Version
+      working-directory: build
+      shell: bash
+      run: |
+        ./launcher --version
+        thisVersion=$(./launcher --version 2>/dev/null | grep "version" | awk '{print $4}')
+        baseVersion="${{ needs.version_baseline.outputs.version }}"
+        if [[ "$thisVersion" != "$baseVersion" ]]; then
+          printf "launcher version %s does not match baseline version %s" "$thisVersion" "$baseVersion"
+          exit 1
+        fi
+
+    - name: Download Osquery
+      working-directory: build
+      run: ./launcher download-osquery --directory .
+
+    - name: Osquery Version
+      working-directory: build
+      run: ./osqueryd --version
```

Comment:
This is probably fine, but it may cause our PRs to fail for very unrelated osquery reasons

Reply:
This is in the existing exec test, so I figured either both should have them or we can remove them in both places